pennylane/ops/cv.py

```diff
@@ -102,6 +102,9 @@ class Rotation(CVOperation):
     def _heisenberg_rep(p):
         return _rotation(p[0])
 
+    def adjoint(self, do_queue=False):
+        return Rotation(-self.parameters[0], self.wires, do_queue=do_queue)
```

Comment:
oh I thought you were going to leave these for another PR 😆

Reply:
I did lol. I think I tried adding some but forgot to take them back out.